doc/ROS2/setup/microROS_Native.md

```diff
@@ -39,7 +39,9 @@ ros2 run micro_ros_setup create_firmware_ws.sh generate_lib
 ```
 
 Copy the files [my_custom_toolchain.cmake](./native_build/my_custom_toolchain.cmake) and [my_custom_colcon.meta](./native_build/my_custom_colcon.meta) in the microros_ws
-
+```bash
+cp ~/DroidControlShip/doc/ROS2/setup/native_build/my_custom_* ~/microros_ws
```

Comment:
This "\~" only works if DCS in directly under $HOME. Some user may not know what "\~" does and fail.
How about 
Example (replace \<DCS-DIR\> with the DroidControlShip workspace location):

`cp <DCS-DIR>/doc/...`

Reply:
Following the tutorial it is located in the home directory. Otherwise in all other places this must be replaced too.

If someone has Linux knowledge, he knows what to do, the others are happy for every clear and simple "just do it" line.